### ISSUE TITLE
IPV6 Geoloc missing.

### ISSUE THREAD

**Issue by gthgame** [TeaSpeak/TeaSpeak#649]
**Describe the bug**
There is no IPV6 geoloc it's always fallback to default country.

**To Reproduce**
Steps to reproduce the behaviour:
 No geoloc for ipv6 always fallback to default country.

**Expected behaviour**
geoloc IP2LocationIPV6.CSS missing, need to be implemented.

**Screenshots**
--

**Logs**
--

**Environment information:**
All

**Additional context**
There is no IPV6 geoloc at this moment. Can you add it ?

**Comment by WolverinDEV** [comment]
Hey,
yes thats right, but I've not yet noticed a large usage for IPv6 addresses.
But it will come :)

**Comment by golmanio** [comment]
Yes it is necessary too tea ******* has already had it for a very long time: it is a functionality that is missing on your software, it must be added as soon as possible.

**Comment by Bluscream** [comment]
:joy:

**Comment by gthgame** [comment]
Any news about ipv6 support ?

**Comment by WolverinDEV** [comment]
No not yet.
